CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+### 2022-04-13
+
+- Consequencer: Moved self.reset_timeout = 500 to a better place
+- Consequencer: Reduced latency by removing the if self.clock_step < 128 check
+- Consequencer: You can now add patterns longer than 32 steps without breaking anything 🙂
+- Consequencer: Added new patterns inspired by African beats and one based on the Fibonacci sequence
+- Consequencer: Added a feature to send a clock out of output 4 which is always in time with the Consequencer. I found this useful to combat the latency between the clock sent to the Consequencer and the gates that come out. With self.output4isClock set to True you can clock all your other modules using output 4 and they will be in-sync with the Consequencer
+- Consequencer: Added explanation of self.output4isClock to doc
+- CVecorder: Added comments to unused functions which are reserved for future use
+- Cvecorder: doc updates
+- Cvecorder: Added menu handling code
```

Comment:
A few of these entries are probably more detailed than we want here. This doc should be targeted at the user and paint the broad strokes of what they're getting with an upgrade to the new version. (and here 'user' can mean both the user of a specific script or the user of the API)

So I don't think we need any details on code changes, just user visible functionality changes. Also, CVecoder is new in this release, so we just need an entry saying that it was added.